<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActiondelayWindow</class>
 <widget class="QDialog" name="ActiondelayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>220</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Nanum Gothic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>대기시간 추가</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: #EFEFEF;
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frame_main">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>340</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
border:1px solid black;
border-radius: 5px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="button_save">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
      <hintingpreference>PreferVerticalHinting</hintingpreference>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
    </property>
    <property name="text">
     <string>저장</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::InsertText"/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cancel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
      <hintingpreference>PreferVerticalHinting</hintingpreference>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
    </property>
    <property name="text">
     <string>취소</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::InsertText"/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_tip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color:black;</string>
    </property>
    <property name="text">
     <string>다음 동작까지 대기할 시간을 설정하세요</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color:black;</string>
    </property>
    <property name="text">
     <string>대기 시간 :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_delay">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
border-radius: 5px;
padding-left: 3px;</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="placeholderText">
     <string>ex) 0.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color:black;</string>
    </property>
    <property name="text">
     <string>초</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
